src/main/groovy/com/github/mamohr/gradle/deploymentstructure/model/Deployment.groovy

```diff
@@ -25,6 +25,7 @@ class Deployment {
     private Set<DependencyModule> dependencyModules = []
     private Set<Module> excludeModules = [] as Set
     private Set<SubSystem> subSystemExcludes = []
+    private Set<Resource> resources = [] as Set
```

Comment:
I did not really understand why I had to use the `as <Class>` syntax here, but I encountered problems with the sorting of the generated xml tags without this. Guess that was the same problem for line 26 @mamohr ? What effect has it?